titles-for-programmers.md

```diff
@@ -2,7 +2,7 @@
 
 ## Programming Management
 
-On Web and SIP programming teams, Lead Programmers (L4) and Principal Programmers (L5) act as tech leads to Junior - Senior Programmers (L1-3). All Leads and Principals report directly to our CTO. Tech leads are responsible for:
+On Web and [SIP (Security, Infrastructure, and Performance)](https://dev.37signals.com/navigating-personal-information-with-care/) programming teams, Lead Programmers (L4) and Principal Programmers (L5) act as tech leads to Junior - Senior Programmers (L1-3). All Leads and Principals report directly to our CTO. Tech leads are responsible for:
```

Comment:
```suggestion
Lead Programmers (L4) and Principal Programmers (L5) act as tech leads to Junior - Senior Programmers (L1-3). All Leads and Principals report directly to our CTO. Tech leads are responsible for:
```

Reply:
Hi Jeremy, thanks for your feedback. If we remove the term of SIP then in that term will be appeared again in the next paragraphs. I'm sorry I can not mark the change like you do above but here are the sections I mention.

> A tech lead on the SIP team also acts as SIP Coordinator. They're responsible for setting cycle priorities, liaising with other project teams, and maintaining general oversight of team resources.

https://github.com/basecamp/handbook/pull/163/files#diff-baba5fd09241c2887b1af16a21b9e9a8d0c47dcb45b17de28e578d367f995e85L12


**In the Individual Contributor Expectations > Judgement (Skill) category**

> | *Judgment* | Demonstrates curiosity in their craft, prioritizes their own development to better serve their team. Researches what's been done in the past and why, attempts to solve problems independently. | Proactively identifies and resolves problems, improves areas of the code they work on, and leaves things in better shape than they found them. Pays attention to detail. | Has accumulated sufficient experience to be consistently sought by peers, across multiple teams, for advice. Improves existing professional standards for the team. | Has the experience, trust and respect to operate at the tech lead level. Defines professional standards. Is sought for advice across disciplines. | Is seen as a leader across teams, technically and culturally. Capable of setting and directing across an entire discipline, such as SIP, Web or Mobile. Is known company-wide as a go-to expert for advice on critical work across disciplines. |


https://github.com/basecamp/handbook/pull/163/files#diff-baba5fd09241c2887b1af16a21b9e9a8d0c47dcb45b17de28e578d367f995e85L23